tests/RustTest.sh

```diff
@@ -41,3 +41,9 @@ else
 fi
 
 cargo bench $TARGET_FLAG
+
+# make sure that the no_std variant builds
+rustup install nightly
```

Comment:
@jrobsonchase `RustTest.sh` isn't for installing Rust, we leave that up to the user. The place I'm looking for no-std tests are in a new Dockerfile, in that directory I linked :-)